tests/test_poincare_math.py

```diff
@@ -370,3 +370,17 @@ def test_distance2plane(a, c):
     dist = poincare.math.dist2plane(z, a, vr, c=c)
 
     np.testing.assert_allclose(dist, dist1, atol=1e-5, rtol=1e-5)
+
+
+def test_klein_transition(a, c):
+    kl = poincare.math.poincare2klein(a, c=c)
+    b = poincare.math.klein2poincare(kl, c=c)
+    np.testing.assert_allclose(a, b, atol=1e-6, rtol=1e-6)
+
+
+def test_midpoint(a, c):
+    # test the midpoint with zero curvature
```

Comment:
@ferrine this test doesn't pass. `poincare.math.midpoint` output is `mids_b = 10 * mids_a`.